org.alloytools.alloy.core/src/main/java/org/alloytools/graphics/util/AlloyGraphics.java

```diff
@@ -0,0 +1,34 @@
+package org.alloytools.graphics.util;
+
+import java.awt.Font;
+import java.awt.GraphicsEnvironment;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+public class AlloyGraphics {
+
+    private static Set<String> availableFontNames;
+
+    public static synchronized String matchBestFontName(String fontName) {
```

Comment:
add doc comment explaining the heuristic this method implements and the meaning of the magic "$" prefix